crates/nu-command/src/database/commands/into_sqlite.rs

```diff
@@ -19,21 +21,23 @@ impl Command for IntoSqliteDb {
 
     fn signature(&self) -> Signature {
         Signature::build("into sqlite")
-            .input_output_types(vec![(Type::Any, Type::Nothing)])
+            .category(Category::Conversions)
+            .input_output_types(vec![
+                (Type::Table(vec![]), Type::Nothing),
+                (Type::Record(vec![]), Type::Nothing),
```

Comment:
Under which circumstances is the `record ->` signature necessary? It may be worth including an example.

Reply:
This is for if you'd want to insert a single record. I added an example.

Reply:
Awesome thanks!